src/index.js

```diff
@@ -0,0 +1,46 @@
+//6
+let user = 'John Doe';
+console.log(user);
+let student = 'Darya';
+console.log(student);
+user = student; // user = 'Darya';
+console.log(user);
+
+//7
+let test = 1;
+test += 1;
+test += '1'; // test = '21'
+console.log(test);
+test = Boolean(test); // true
+console.log(test);
+
+//8
+{
+  console.log('Multiplication of elements in array');
+  let result = 1;
+  let arr = [2, 3, 5, 8];
+  for (let i = 0; i < arr.length; i++) {
+    result *= arr[i];
+  }
+  console.log(result);
```

Comment:
Just advice. You can write like this:
`console.log('Multiplication of elements in array', result)`